### PR TITLE
Workflows

### PR DESCRIPTION
**Pull request by Tonnu**
New docs, updated with workflows editor changes

### CODE REVIEW

File: content/docs/workflows/chaining.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
-A major benefit introduced By Workflows is that you can now chain as many
+A major benefit introduced by Workflows is that you can now chain as many
 pipelines as you want, instead of just **build** -> **deploy**.
 
-Chaining Workflows can be done up to 10 times to create the powerful, custom
+Chaining pipelines can be done up to 10 times to create the powerful, custom
 Workflow that you need.
 
 In the screenshot below you can see three Workflows chained together.
```
Comment (davegri) [inline content/docs/workflows/chaining.md]: Workflows > pipelines

File: content/docs/workflows/chaining.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-Inspecting the **build** workflow you can see it gets triggered based on a `git push`.
+First, we build our base-image, via the **inital-build** pipeline.
+Inspecting the **intial-build** pipeline you can see it gets triggered based on a `git push`.
 
-![image](/images/githook.png)
+![image](/images/create-pipeline.png)
 
-Next, we run our unit test through the **run-tests** Workflow. This Workflow is
-triggered automatically after the **build** Workflow has successfully finished.
+Next, we run our unit test through the **tests** Workflow. This Workflow is
```
Comment (davegri) [inline content/docs/workflows/chaining.md]: Workflow > pipeline

File: content/docs/workflows/chaining.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+triggered automatically after the **initial-build** Workflow has successfully finished.
```
Comment (davegri) [inline content/docs/workflows/chaining.md]: Workflow > pipeline

File: content/docs/workflows/chaining.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
 The screenshot below showcases how this Workflow is chained.
```
Comment (davegri) [inline content/docs/workflows/chaining.md]: Workflow > pipeline

File: content/docs/workflows/chaining.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
 ## Chaining Workflows
```
Comment (davegri) [inline content/docs/workflows/chaining.md]: Chaining Pipelines

File: content/docs/workflows/chaining.md [hunk 6]
```diff
@@ -1,26 +1,26 @@
...
-triggered automatically after the **build** Workflow has successfully finished.
+Next, we run our unit test through the **tests** Workflow. This Workflow is
+triggered automatically after the **initial-build** Workflow has successfully finished.
 The screenshot below showcases how this Workflow is chained.
 
-![image](/images/chainbuild.png)
+![image](/images/chaining.png)
 
-The **run-tests** Workflow is triggered on any passing **build** on *any* branch.
+The **tests** Workflow is triggered on any passing **build** on *any* branch.
```
Comment (davegri) [inline content/docs/workflows/chaining.md]: Workflow > pipeline

File: content/docs/workflows/managing.md [hunk 7]
```diff
@@ -2,87 +2,72 @@
...
 top right of your application page. Alternatively, you can go to your
-application **settings** page and head to the **Workflows** section. 
+application **settings** page and head to the **Workflows** section.
 
-To create a workflow you must:
+Workflows can be created by chaining pipelines together. To do that, you must first specify these pipelines in the UI.
+To create a pipeline, you must:
 
 1. Give it a name e.g.: “**deploy-to-dockerhub**”;
 2. Decide how this workflow gets executed; This can be a Git or another Workflow hook;
```
Comment (davegri) [inline content/docs/workflows/managing.md]: Workflow > pipeline

File: content/docs/workflows/managing.md [hunk 8]
```diff
@@ -2,87 +2,72 @@
...
 3. Specify which pipeline you would like to execute; the pipeline-name
 corresponds to the pipeline-name as you defined it in your wercker.yml;
 
+![image](/images/pipelines-ui.png)
+
 Note that, when creating an application on the Wercker platform, a `build`
 Worfklow, which contains a Git hook and executes a pipeline called `build`,
```
Comment (davegri) [inline content/docs/workflows/managing.md]: Worfklow > pipeline

File: content/docs/workflows/managing.md [hunk 9]
```diff
@@ -2,87 +2,72 @@
...
 will be automatically created.
 
 ### Updating Workflows
```
Comment (davegri) [inline content/docs/workflows/managing.md]: Updating Pipelines

File: content/docs/workflows/managing.md [hunk 10]
```diff
@@ -2,87 +2,72 @@
...
 4. Admin
 
-**Public**:  If both your application and Workflow are set to **public** everyone will be able to see the runs executed for that Workflow. If your application is not public, all your collaborators (regardless of their permissions) will be able to see runs executed for that Workflow.
-
-**View runs:** Allow only collaborators with ‘view runs’ permissions to manage this target
-
-**View Runs + Manage workflows**: Allow only collaborators with ‘View runs + Manage workflows’ to manage this target
-
-**Admin**: Only administrator collaborators can manage this Workflow
+**Public**:  If both your application and pipeline are set to **public** everyone will be able to see the runs executed for that pipeline. If your application is not public, all your collaborators (regardless of their permissions) will be able to see runs executed for that Workflow.
```
Comment (davegri) [inline content/docs/workflows/managing.md]: Workflow > Pipeline

File: content/docs/workflows/managing.md [hunk 11]
```diff
@@ -2,87 +2,72 @@
...
 
+**View runs:** Allow only collaborators with ‘view runs’ permissions to manage this pipelien
```
Comment (davegri) [inline content/docs/workflows/managing.md]: Execute pipelines

File: content/docs/workflows/managing.md [hunk 12]
```diff
@@ -2,87 +2,72 @@
...
 
-### Deleting Workflows
+**View Runs + Manage pipelines**: Allow only collaborators with ‘View runs + Manage workflows’ to manage this pipeline
```
Comment (davegri) [inline content/docs/workflows/managing.md]: Execute & manage pipelines

File: content/docs/workflows/managing.md [hunk 13]
```diff
@@ -2,87 +2,72 @@
...
-You can reset which Hook type your Workflow listens on, but keep in mind that
-this will completely remove any Hook-specific settings you might have.
-
-For Workflow-based hooks you can specify any amount of Workflows that this
-Workflow should listen on. For Git-based hooks you can specify on which branch
-this Workflow should start.
+#### Pipeline permissions
 
-Finally each target can have one of the following permission requirements:
+Each pipeline can have one of the following permission requirements:
```
Comment (davegri) [inline content/docs/workflows/managing.md]: Permission names same as other comment

**Comment by Tonnu** [comment]
Pushed my changes to #400 by accident. Will rebase that branch on this one and work from there. All your suggestions are in there @davegri , thanks 👍

**Pull request closed without merging**